app/src/main/java/com/willowtree/vocable/presets/PresetsViewModel.kt

```diff
@@ -23,7 +25,7 @@ class PresetsViewModel : ViewModel(), KoinComponent {
         const val CATEGORY_MY_SAYINGS = "My Sayings"
     }
 
-    private val sharedPrefs: VocableSharedPreferences by inject()
+    private val presetsRepository: PresetsRepository by inject()
```

Comment:
You should definitely check out using viewModel injection that Koin is offering.